src/app/personside/infotabs/InfoTabs.tsx

```diff
@@ -70,11 +70,11 @@ function InfoTabs(props: Props) {
                             path={basePath + INFOTABS.UTBETALING + '/'}
                             component={() => <UtbetalingerContainer />}
```

Comment:
Tar du med denne også? Bare for å slippe fremtidige issues

Reply:
Ooops, den slapp visst gjennom 👀